base/shell/cmd/cmd.c

```diff
@@ -1423,6 +1423,12 @@ ReadLine(TCHAR *commandline, BOOL bMore)
             ConOutChar(_T('\n'));
             return FALSE;
         }
+
+        if (readline[0] == L'\0')
```

Comment:
cmd uses TCHARs so this should be _T('\0')

Reply:
Good catch.  I found another stray L in the sources as well, so I'll fix that while I'm at it.